monitor/price/pricemonitoring.go

```diff
@@ -258,9 +258,10 @@ func (e *Engine) CheckPrice(ctx context.Context, as AuctionState, trades []*type
 		e.resetPriceHistory(trades)
 		e.initialised = true
 	}
+	inAuction := as.InAuction()
 	// market is not in auction, or in batch auction
-	if fba := as.IsFBA(); !as.InAuction() || fba {
-		bounds := e.checkBounds(ctx, trades)
+	if fba := as.IsFBA(); !inAuction || fba {
+		bounds := e.checkBounds(ctx, trades, inAuction)
```

Comment:
This won't work once we have FBA's -> `inAuction` will then always be `true`, even if (when `CheckPrice` is called is likely to be the case), we've reached the end of a batch.

Reply:
hmm ok, not sure why we're anticipating it already if it's not really on the immediate horizon, but I guess since we started doing it we might as well carry on, will fix.

Reply:
Fixed now I believe.